index.js

```diff
@@ -244,13 +244,38 @@ module.exports = {
 				// Arrow functions can simplify tests
 				'react/jsx-no-bind': 'off',
 
+				// https://github.com/jest-community/eslint-plugin-jest
+				'jest/no-conditional-expect': 'error',
+				'jest/no-identical-title': 'error',
+				'jest/no-interpolation-in-snapshots': 'error',
+				'jest/no-jasmine-globals': 'error',
+				'jest/no-jest-import': 'error',
+				'jest/no-mocks-import': 'error',
+				'jest/valid-describe-callback': 'error',
+				'jest/valid-expect': 'error',
+				'jest/valid-expect-in-promise': 'error',
+				'jest/valid-title': 'warn',
+
 				// https://github.com/testing-library/eslint-plugin-testing-library
 				'testing-library/await-async-query': 'error',
 				'testing-library/await-async-utils': 'error',
-				'testing-library/no-await-sync-query': 'warn',
+				'testing-library/no-await-sync-query': 'error',
+				'testing-library/no-container': 'error',
+				'testing-library/no-debugging-utils': 'error',
 				'testing-library/no-dom-import': ['error', 'react'],
+				'testing-library/no-node-access': 'off',
```

Comment:
Can we just remove it? Should we "off" this?

Reply:
Either is fine, let's remove it.